bin/polylint.js

```diff
@@ -170,6 +171,29 @@ process.on('unhandledRejection', function(reason, p) {
   fatalFailureOccurred = true;
 });
 
+// Find the bower dir.
+var searchPath = options.root;
+var parentDirs = [];
+var foundBower = false;
+while (!foundBower) {
+  var maybeBower = path.resolve.apply(undefined, [searchPath].concat(parentDirs).concat([options.bowerdir]));
+  if (maybeBower == path.join('/', options.bowerdir)) {
+    break;
+  }
+  try {
+    stat = fs.statSync(maybeBower);
+    foundBower = true;
+  } catch (err) {
+    parentDirs.push('..');
+  }
+}
+if (!foundBower) {
+  options.bowerdir = undefined;
+} else {
+  options.bowerdir = path.join(path.join(parentDirs), options.bowerdir);
```

Comment:
This is kinda unclear, you're using `options.bowerdir` to find `options.bowerdir`?

I guess it's initally a partial path, like 'paper-button/bower.json' and you're walking up the tree of parents?